<launch>


	<arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->

	<!-- Set parameters -->
	<group ns="cam_L">
		<node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="apriltag" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)" respawn="true">
			<rosparam command="load" file="$(find tm_apriltag)/config/mLDR/settings.yaml" ns="" />
			<rosparam command="load" file="$(find tm_apriltag)/config/mLDR/tags.yaml" ns="" />

			<param name="camera_frame" type="str" value="usb_camera_L"  />
			<param name="publish_tag_detections_image" type="bool" value="False" />
			<!-- <remap from="/tag_detections_image" to="/tag_detections_image_L" />
			<remap from="/tag_detections" to="/tag_detections_L" /> -->
			<remap from="image_rect" to="image_raw" />
			<remap from="camera_info" to="camera_info" />
			<!-- the following param cannot be dynamic changed as it only be read by init-->
			<param name="gray_threshold" type="int" value="1" />	<!-- added in the library: continous node-->
			<!-- 0: No threshold 1: All use threshold 2: switch grey and threshold  -->
			<param name="use_gray_mode_" type="int" value="0" />
			<param name="publish_landmarks" type="bool" value="true" />
		</node>
	</group>

</launch>


